src/Native/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -334,7 +334,7 @@ int UTF8ToWideChar(const char* bytes, int len, wchar_t* buffer, int bufLen)
     size_t inbufbytesleft = len;
     size_t outbufbytesleft = bufLen;
 
-    int rc = iconv(cd, &inbuf, &inbufbytesleft, &outbuf, &outbufbytesleft);
+    int rc = iconv(cd, const_cast<const char **>(&inbuf), &inbufbytesleft, &outbuf, &outbufbytesleft);
```

Comment:
The usual way to address this would be via a config switch. However, I would rather get rid of these functions completely. They are just compensating for Win32-isms that were not cleaned up yet. I am looking into what would be the best way to do it.